BluePic-Server/Package.swift

```diff
@@ -19,14 +19,14 @@ import PackageDescription
 let package = Package(
   name: "BluePic-Server",
   dependencies: [
-   .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 0, minor: 22),
-    .Package(url: "https://github.com/IBM-Swift/Kitura-CouchDB.git", majorVersion: 0, minor: 22),
-    .Package(url: "https://github.com/IBM-Swift/Swift-cfenv.git", majorVersion: 1, minor: 4),
+   .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 0, minor: 31),
```

Comment:
`.Package(url: "https://github.com/ibm-bluemix-mobile-services/bluemix-simple-http-client-swift.git", majorVersion: 0, minor: 5),` This package uses Kitura-net 0.31. Should I ping Anton about this?